packages/cli/test/me.endpoints.test.ts

```diff
@@ -0,0 +1,531 @@
+import { hashSync, genSaltSync } from 'bcryptjs';
+import express = require('express');
+import { getConnection } from 'typeorm';
+import validator from 'validator';
+import { v4 as uuid } from 'uuid';
+
+import config = require('../config');
+import * as utils from './shared/utils';
+import { SUCCESS_RESPONSE_BODY } from './shared/constants';
+import { Db } from '../src';
+import { User } from '../src/databases/entities/User';
+
+describe('/me endpoints', () => {
+	describe('Shell requests', () => {
+		let app: express.Application;
+
+		beforeAll(async () => {
+			app = utils.initTestServer({ me: true });
+			await utils.initTestDb();
+			await utils.truncateUserTable();
+		});
+
+		beforeEach(async () => {
+			const role = await Db.collections.Role!.findOneOrFail({ name: 'owner', scope: 'global' });
+
+			await Db.collections.User!.save({
+				id: uuid(),
+				firstName: 'default',
+				lastName: 'default',
+				createdAt: new Date(),
+				updatedAt: new Date(),
+				globalRole: role,
+			});
+		});
+
+		afterEach(async () => {
+			await utils.truncateUserTable();
+		});
+
+		afterAll(() => {
+			return getConnection().close();
+		});
+
+		test('GET /me should return sanitized shell', async () => {
+			const shell = await Db.collections.User!.findOneOrFail();
+			const shellAgent = await utils.createAgent(app, shell);
+
+			const response = await shellAgent.get('/me');
+
+			expect(response.statusCode).toBe(200);
+
+			const {
+				id,
+				email,
+				firstName,
+				lastName,
+				personalizationAnswers,
+				globalRole,
+				password,
+				resetPasswordToken,
+			} = response.body.data;
+
+			expect(validator.isUUID(id)).toBe(true);
+			expect(email).toBeNull();
+			expect(firstName).toBe('default');
+			expect(lastName).toBe('default');
```

Comment:
we are moving away from the `'default'` input there
```suggestion
			expect(firstName)).toBeNull();
			expect(lastName)).toBeNull();
```